docs/book/src/tasks/installation.md

```diff
@@ -243,38 +77,117 @@ stringData:
 
 {{#/tab }}
 {{#tab OpenStack}}
 
-Check the [OpenStack provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/releases) for an up-to-date components file.
+Please visit the [OpenStack getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/docs/getting-started.md).
 
-For more detailed information, e.g. about prerequisites visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/docs/getting-started.md).
+{{#/tab }}
+{{#tab Metal3}}
 
-```bash
-kubectl create -f {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-openstack" asset:"infrastructure-components.yaml" version:"0.2.x"}}
+Please visit the [Metal3 getting started guide](https://github.com/metal3-io/cluster-api-provider-metal3/blob/master/docs/getting-started.md).
+
+{{#/tab }}
+{{#/tabs }}
+
+## Install clusterctl
+The clusterctl CLI tool handles the lifecycle of a Cluster API management cluster.
+
+{{#tabs name:"install-clusterctl" tabs:"linux,macOS"}}
+{{#tab linux}}
+
+#### Install clusterctl binary with curl on linux
+Download the latest release; for example, to download version v0.3.0 on linux, type:
+```
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.0/clusterctl-linux-amd64 -o clusterctl
+```
+Make the kubectl binary executable.
+```
+chmod +x ./clusterctl
+```
+Move the binary in to your PATH.
+```
+sudo mv ./clusterctl /usr/local/bin/clusterctl
+```
+Test to ensure the version you installed is up-to-date:
+```
+clusterctl version
 ```
 
+{{#/tab }}
+{{#tab macOS}}
+
+#### Install clusterctl binary with curl on macOS
+Download the latest release; for example, to download version v0.3.0 on macOS, type:
+```
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.0/clusterctl-darwin-amd64 -o clusterctl
+```
+Make the kubectl binary executable.
+```
+chmod +x ./clusterctl
+```
+Move the binary in to your PATH.
+```
+sudo mv ./clusterctl /usr/local/bin/clusterctl
+```
+Test to ensure the version you installed is up-to-date:
+```
+clusterctl version
+```
 {{#/tab }}
 {{#/tabs }}
 
-## Custom images
+## Install and/or configure a kubernetes cluster
+
+Cluster API requires an existing kubernetes cluster accessible via kubectl; during the installation process the 
+Kubernetes cluster will be transformed into a [management cluster] by installing the Cluster API [provider components], so it 
+is recommended to keep it separated from any application workload.
 
-Cluster API relies on pre-built images with Kubernetes components preinstalled. The [image builder](https://image-builder.sigs.k8s.io/) project has requirements and examples on how to create and publish images.
+Please note that it is a common practice the creation of a temporary [bootstrap cluster] used to provision 
+a target [management cluster] on the selected [infrastructure provider], e.g on AWS.
```

Comment:
```suggestion
a target [management cluster] on the selected [infrastructure provider].
```
Not sure we need the AWS example here, there's already a link to the definition of infra provider...